src/main/java/teammates/logic/core/GoogleCloudLoggingService.java

```diff
@@ -463,28 +459,14 @@ public LogSearchParams setOrder(String order) {
      */
     private static class PageParams {
```

Comment:
If `PageParams` contains only one field without any logic, we might as well just use that field instead of needlessly wrapping around this class.

Reply:
This class is removed